lib/jsonArray.js

```diff
@@ -0,0 +1,34 @@
+const assert = require('assert');
+
+class JsonArray {
+  constructor(options = {}) {
+    assert.ok(!!options.value, 'JsonArray requires a value');
+    this.options = options;
+  }
+
+  get length() {
+    return 1;
```

Comment:
not sure what this is doing

Reply:
@asulaiman we assert that the length of all arrays in our template is 1 here https://github.com/hubba/jyson/pull/36/files#diff-939716d327873b484c0914e8a74fc40bR72, so this makes the `JsonArray` object have a length of 1.

It is a little odd, open to better ideas.